python/ray/util/ml_utils/BUILD

```diff
@@ -10,6 +10,14 @@ py_test(
     deps = [":ml_util_lib"]
```

Comment:
In a separate PR, could we get rid of ml_utils and move it into the ml folder?

Reply:
We can move this back to Tune as soon as we consolidated callbacks (and checkpoint management) - this on our backlog and will happen soon.